packages/internal/test/upgrade-api.test.js

```diff
@@ -18,3 +21,15 @@ test('isUpgradeDisconnection must recognize original-format disconnection object
   });
   t.true(isUpgradeDisconnection(disconnection));
 });
+
+test('isAbandonedError recognizes marshalled vat terminated errors', t => {
+  const { fromCapData, toCapData } = makeMarshal(undefined, undefined, {
+    serializeBodyFormat: 'smallcaps',
+    errorIdNum: 70_000,
+    marshalSaveError: () => {},
+  });
+  const error = harden(Error('vat terminated'));
+  const remoteError = fromCapData(toCapData(error));
+
+  t.true(isAbandonedError(remoteError));
```

Comment:
Nice. But to reiterate, this probably depends on stability beyond what I expect to propose for ocapn to mandate.